crypto/box/src/hkdf_box.rs

```diff
@@ -107,15 +108,15 @@ where
         // KDF
         let (aes_key, aes_nonce) = Self::kdf_step(&shared_secret);
 
+        let ct_aes_nonce =
+            <GenericArray<u8, AeadAlgo::NonceSize>>::from_slice(aes_nonce.as_slice());
```

Comment:
it's a bit confusing that we have to do this, is it still needed at this revision? it looks to me that `aes_nonce` is already of this type

Reply:
Fixed, not needed anymore.